src/backend/joanie/core/api/admin/__init__.py

```diff
@@ -689,6 +691,31 @@ def refund(self, request, pk=None):  # pylint:disable=unused-argument
 
         return Response(status=HTTPStatus.ACCEPTED)
 
+    @extend_schema(
+        request=None,
+        responses={
+            (200, "text/csv"): OpenApiTypes.OBJECT,
+            404: serializers.ErrorResponseSerializer,
+        },
+    )
+    @action(methods=["GET"], detail=False)
+    def export(self, request):
+        """
+        Export orders to a CSV file.
+        """
+        queryset = self.filter_queryset(self.get_queryset()).order_by("created_on")
```

Comment:
You chose to export by ascending order?

Reply:
I believe this was for debugging purpose, as the ordering in the model is already `-created_on`

Reply:
yes, but the viewset `OrderViewSet` is set to : 
`ordering_fields = ["created_on"]`